diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7eda55949..43baf612f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -146,7 +146,7 @@ endif()
 #-----------------------------------------------------------------------------
 
 set(EXECUTABLE_OUTPUT_PATH ${PROJECT_BINARY_DIR}/bin)
-set(LIBRARY_OUTPUT_PATH ${PROJECT_BINARY_DIR}/lib64)
+set(LIBRARY_OUTPUT_PATH ${PROJECT_BINARY_DIR}/lib)
 
 #-----------------------------------------------------------------------------
 
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 860cfdb87..45b24ac3c 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -527,7 +527,7 @@ configure_file("${PROJECT_SOURCE_DIR}/cmake/ChronoConfig.cmake.in"
 if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
     set(CH_CONFIG_INSTALL_PATH "cmake")
 else()
-    set(CH_CONFIG_INSTALL_PATH "lib64/cmake")
+    set(CH_CONFIG_INSTALL_PATH "lib/cmake")
 endif()
 
 install(FILES "${PROJECT_BINARY_DIR}/cmake/ChronoConfig.cmake.install"
diff --git a/src/chrono/CMakeLists.txt b/src/chrono/CMakeLists.txt
index 97c4f5e12..e5f2764c1 100644
--- a/src/chrono/CMakeLists.txt
+++ b/src/chrono/CMakeLists.txt
@@ -1043,8 +1043,8 @@ endif()
 # Install the main ChronoEngine library
 install(TARGETS ChronoEngine
     RUNTIME DESTINATION bin
-    LIBRARY DESTINATION lib64
-    ARCHIVE DESTINATION lib64)
+    LIBRARY DESTINATION lib
+    ARCHIVE DESTINATION lib)
 
 # Install all header files for the main ChronoEngine library 
 install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/
diff --git a/src/chrono_cascade/CMakeLists.txt b/src/chrono_cascade/CMakeLists.txt
index bb79799bc..132e03c3d 100644
--- a/src/chrono_cascade/CMakeLists.txt
+++ b/src/chrono_cascade/CMakeLists.txt
@@ -117,8 +117,8 @@ ADD_DEPENDENCIES(ChronoEngine_cascade ChronoEngine)
 
 INSTALL(TARGETS ChronoEngine_cascade
     RUNTIME DESTINATION bin
-    LIBRARY DESTINATION lib64
-    ARCHIVE DESTINATION lib64)
+    LIBRARY DESTINATION lib
+    ARCHIVE DESTINATION lib)
 
 INSTALL(FILES ${ChronoEngine_CASCADE_HEADERS} DESTINATION include/chrono_cascade)
 
diff --git a/src/chrono_cosimulation/CMakeLists.txt b/src/chrono_cosimulation/CMakeLists.txt
index 81b604c9b..c002ec112 100644
--- a/src/chrono_cosimulation/CMakeLists.txt
+++ b/src/chrono_cosimulation/CMakeLists.txt
@@ -84,8 +84,8 @@ ADD_DEPENDENCIES (ChronoEngine_cosimulation ChronoEngine)
 	
 INSTALL(TARGETS ChronoEngine_cosimulation
         RUNTIME DESTINATION bin
-        LIBRARY DESTINATION lib64
-        ARCHIVE DESTINATION lib64)		
+        LIBRARY DESTINATION lib
+        ARCHIVE DESTINATION lib)		
 
 INSTALL(FILES ${ChronoEngine_COSIMULATION_HEADERS}
         DESTINATION include/chrono_cosimulation)
diff --git a/src/chrono_fea/CMakeLists.txt b/src/chrono_fea/CMakeLists.txt
index d52cfafe2..979341842 100644
--- a/src/chrono_fea/CMakeLists.txt
+++ b/src/chrono_fea/CMakeLists.txt
@@ -149,8 +149,8 @@ ADD_DEPENDENCIES(ChronoEngine_fea ChronoEngine)
 
 INSTALL(TARGETS ChronoEngine_fea
         RUNTIME DESTINATION bin
-        LIBRARY DESTINATION lib64
-        ARCHIVE DESTINATION lib64)
+        LIBRARY DESTINATION lib
+        ARCHIVE DESTINATION lib)
 
 INSTALL(FILES ${ChronoEngine_FEA_HEADERS}
         DESTINATION include/chrono_fea)
diff --git a/src/chrono_fsi/CMakeLists.txt b/src/chrono_fsi/CMakeLists.txt
index 01a78671b..d2c217fa9 100644
--- a/src/chrono_fsi/CMakeLists.txt
+++ b/src/chrono_fsi/CMakeLists.txt
@@ -173,8 +173,8 @@ TARGET_LINK_LIBRARIES(ChronoEngine_fsi ${LIBRARIES})
 
 INSTALL(TARGETS ChronoEngine_fsi
         RUNTIME DESTINATION bin
-        LIBRARY DESTINATION lib64
-        ARCHIVE DESTINATION lib64)
+        LIBRARY DESTINATION lib
+        ARCHIVE DESTINATION lib)
 
 INSTALL(FILES ${ChronoEngine_FSI_HEADERS}
         DESTINATION include/chrono_fsi)
diff --git a/src/chrono_irrlicht/CMakeLists.txt b/src/chrono_irrlicht/CMakeLists.txt
index e699e16c1..e5cf7363e 100644
--- a/src/chrono_irrlicht/CMakeLists.txt
+++ b/src/chrono_irrlicht/CMakeLists.txt
@@ -174,8 +174,8 @@ ENDIF()
 
 INSTALL(TARGETS ChronoEngine_irrlicht
   RUNTIME DESTINATION bin
-  LIBRARY DESTINATION lib64
-  ARCHIVE DESTINATION lib64
+  LIBRARY DESTINATION lib
+  ARCHIVE DESTINATION lib
 )
 
 INSTALL(FILES ${ChronoEngine_IRRLICHT_HEADERS} DESTINATION include/chrono_irrlicht)
diff --git a/src/chrono_matlab/CMakeLists.txt b/src/chrono_matlab/CMakeLists.txt
index 4f23c1ee6..3e014748c 100644
--- a/src/chrono_matlab/CMakeLists.txt
+++ b/src/chrono_matlab/CMakeLists.txt
@@ -81,8 +81,8 @@ ADD_DEPENDENCIES(ChronoEngine_matlab ChronoEngine)
 
 INSTALL(TARGETS ChronoEngine_matlab
         RUNTIME DESTINATION bin
-        LIBRARY DESTINATION lib64
-        ARCHIVE DESTINATION lib64)       
+        LIBRARY DESTINATION lib
+        ARCHIVE DESTINATION lib)       
 
 INSTALL(FILES ${ChronoEngine_MATLAB_HEADERS}
-        DESTINATION include/chrono/unit_MATLAB)
\ No newline at end of file
+        DESTINATION include/chrono/unit_MATLAB)
diff --git a/src/chrono_mkl/CMakeLists.txt b/src/chrono_mkl/CMakeLists.txt
index 79118b38c..f1d657263 100644
--- a/src/chrono_mkl/CMakeLists.txt
+++ b/src/chrono_mkl/CMakeLists.txt
@@ -101,8 +101,8 @@ TARGET_LINK_LIBRARIES(ChronoEngine_mkl
 
 INSTALL(TARGETS ChronoEngine_mkl
         RUNTIME DESTINATION bin
-        LIBRARY DESTINATION lib64
-        ARCHIVE DESTINATION lib64)
+        LIBRARY DESTINATION lib
+        ARCHIVE DESTINATION lib)
 
 INSTALL(FILES ${ChronoEngine_MKL_HEADERS} 
         DESTINATION include/chrono_mkl)
diff --git a/src/chrono_ogre/CMakeLists.txt b/src/chrono_ogre/CMakeLists.txt
index 0b41a2587..f04db48b6 100644
--- a/src/chrono_ogre/CMakeLists.txt
+++ b/src/chrono_ogre/CMakeLists.txt
@@ -128,7 +128,7 @@ TARGET_LINK_LIBRARIES(ChronoEngine_ogre ${LIBRARIES})
 
 INSTALL(TARGETS ChronoEngine_ogre
 	RUNTIME DESTINATION bin
-	LIBRARY DESTINATION lib64
-	ARCHIVE DESTINATION lib64
+	LIBRARY DESTINATION lib
+	ARCHIVE DESTINATION lib
 	)
 	
diff --git a/src/chrono_opengl/CMakeLists.txt b/src/chrono_opengl/CMakeLists.txt
index 0d57ce22f..c6ea45e5a 100644
--- a/src/chrono_opengl/CMakeLists.txt
+++ b/src/chrono_opengl/CMakeLists.txt
@@ -291,8 +291,8 @@ TARGET_LINK_LIBRARIES(ChronoEngine_opengl ${CE_OpenGL_LIBRARIES})
 
 INSTALL(TARGETS ChronoEngine_opengl
   RUNTIME DESTINATION bin
-  LIBRARY DESTINATION lib64
-  ARCHIVE DESTINATION lib64)
+  LIBRARY DESTINATION lib
+  ARCHIVE DESTINATION lib)
 
 INSTALL(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/
     DESTINATION include/chrono_opengl
diff --git a/src/chrono_parallel/CMakeLists.txt b/src/chrono_parallel/CMakeLists.txt
index e6ecd1c1d..e14e016b7 100644
--- a/src/chrono_parallel/CMakeLists.txt
+++ b/src/chrono_parallel/CMakeLists.txt
@@ -466,8 +466,8 @@ TARGET_LINK_LIBRARIES(ChronoEngine_parallel ${CHRONO_PARALLEL_LINKED_LIBRARIES})
 
 INSTALL(TARGETS ChronoEngine_parallel
         RUNTIME DESTINATION bin
-        LIBRARY DESTINATION lib64
-        ARCHIVE DESTINATION lib64)
+        LIBRARY DESTINATION lib
+        ARCHIVE DESTINATION lib)
 
 INSTALL(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/
         DESTINATION include/chrono_parallel
diff --git a/src/chrono_postprocess/CMakeLists.txt b/src/chrono_postprocess/CMakeLists.txt
index 04f86e0f4..459b0963f 100644
--- a/src/chrono_postprocess/CMakeLists.txt
+++ b/src/chrono_postprocess/CMakeLists.txt
@@ -56,8 +56,8 @@ ADD_DEPENDENCIES(ChronoEngine_postprocess ChronoEngine)
 	
 INSTALL(TARGETS ChronoEngine_postprocess
         RUNTIME DESTINATION bin
-        LIBRARY DESTINATION lib64
-        ARCHIVE DESTINATION lib64)		
+        LIBRARY DESTINATION lib
+        ARCHIVE DESTINATION lib)		
 
 INSTALL(FILES ${ChronoEngine_POSTPROCESS_HEADERS}
         DESTINATION include/chrono_postprocess)
diff --git a/src/chrono_python/CMakeLists.txt b/src/chrono_python/CMakeLists.txt
index 2a689b41a..34e38025b 100644
--- a/src/chrono_python/CMakeLists.txt
+++ b/src/chrono_python/CMakeLists.txt
@@ -86,8 +86,8 @@ ADD_DEPENDENCIES(ChronoEngine_pyparser ChronoEngine)
 
 install(TARGETS ChronoEngine_pyparser
         RUNTIME DESTINATION bin
-        LIBRARY DESTINATION lib64
-        ARCHIVE DESTINATION lib64)
+        LIBRARY DESTINATION lib
+        ARCHIVE DESTINATION lib)
 
 #-----------------------------------------------------------------------------
 # Find SWIG
diff --git a/src/chrono_vehicle/CMakeLists.txt b/src/chrono_vehicle/CMakeLists.txt
index dbe9abd73..d15980e08 100644
--- a/src/chrono_vehicle/CMakeLists.txt
+++ b/src/chrono_vehicle/CMakeLists.txt
@@ -812,8 +812,8 @@ target_link_libraries(ChronoEngine_vehicle ${LIBRARIES})
 
 install(TARGETS ChronoEngine_vehicle
         RUNTIME DESTINATION bin
-        LIBRARY DESTINATION lib64
-        ARCHIVE DESTINATION lib64)
+        LIBRARY DESTINATION lib
+        ARCHIVE DESTINATION lib)
 
 install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/
         DESTINATION include/chrono_vehicle
diff --git a/src/demos/CMakeLists.txt b/src/demos/CMakeLists.txt
index 744783501..9e42b20f8 100644
--- a/src/demos/CMakeLists.txt
+++ b/src/demos/CMakeLists.txt
@@ -8,16 +8,16 @@ set(CMAKE_SKIP_BUILD_RPATH  FALSE)
 # (but later on when installing)
 set(CMAKE_BUILD_WITH_INSTALL_RPATH FALSE)
 
-set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib64")
+set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib")
 
 # add the automatically determined parts of the RPATH
 # which point to directories outside the build tree to the install RPATH
 set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
 
 # the RPATH to be used when installing, but only if it's not a system directory
-list(FIND CMAKE_PLATFORM_IMPLICIT_LINK_DIRECTORIES "${CMAKE_INSTALL_PREFIX}/lib64" isSystemDir)
+list(FIND CMAKE_PLATFORM_IMPLICIT_LINK_DIRECTORIES "${CMAKE_INSTALL_PREFIX}/lib" isSystemDir)
 if("${isSystemDir}" STREQUAL "-1")
-   set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib64")
+   set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib")
 endif("${isSystemDir}" STREQUAL "-1")
 
 #--------------------------------------------------------------
diff --git a/src/tests/unit_tests/CMakeLists.txt b/src/tests/unit_tests/CMakeLists.txt
index d21d79a5b..7c1dd410a 100644
--- a/src/tests/unit_tests/CMakeLists.txt
+++ b/src/tests/unit_tests/CMakeLists.txt
@@ -12,16 +12,16 @@ SET(CMAKE_SKIP_BUILD_RPATH  FALSE)
 # (but later on when installing)
 SET(CMAKE_BUILD_WITH_INSTALL_RPATH FALSE) 
 
-SET(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib64")
+SET(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib")
 
 # add the automatically determined parts of the RPATH
 # which point to directories outside the build tree to the install RPATH
 SET(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
 
 # the RPATH to be used when installing, but only if it's not a system directory
-LIST(FIND CMAKE_PLATFORM_IMPLICIT_LINK_DIRECTORIES "${CMAKE_INSTALL_PREFIX}/lib64" isSystemDir)
+LIST(FIND CMAKE_PLATFORM_IMPLICIT_LINK_DIRECTORIES "${CMAKE_INSTALL_PREFIX}/lib" isSystemDir)
 IF("${isSystemDir}" STREQUAL "-1")
-   SET(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib64")
+   SET(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib")
 ENDIF("${isSystemDir}" STREQUAL "-1")
 
 #--------------------------------------------------------------
diff --git a/template_project/CMakeLists.txt b/template_project/CMakeLists.txt
index 215b2498f..1882b45b4 100644
--- a/template_project/CMakeLists.txt
+++ b/template_project/CMakeLists.txt
@@ -59,7 +59,7 @@ project(my_project)
 # In this example, we only request the Irrlicht module (required)
 #--------------------------------------------------------------
 
-LIST(APPEND CMAKE_PREFIX_PATH "${CMAKE_INSTALL_PREFIX}/../Chrono/lib64")
+LIST(APPEND CMAKE_PREFIX_PATH "${CMAKE_INSTALL_PREFIX}/../Chrono/lib")
 find_package(Chrono
              COMPONENTS Irrlicht
              CONFIG)
